Packs/Infoblox/Integrations/Infoblox/Infoblox.py

```diff
@@ -348,28 +722,61 @@ def get_ip_command(client: Client, args: Dict[str, str]) -> Tuple[str, Dict, Dic
         Outputs
     """
     ip = args.get('ip')
-    try:
-        raw_response = client.get_ip(ip)
-    except DemistoException as e:
-        if e.message and NETWORK_NOT_FOUND in e.message:
-            return "No indicators found", {}, {}
-        else:
-            raise
+    network = args.get('network')
+    from_ip = args.get('from_ip')
+    to_ip = args.get('to_ip')
+
+    # Input validation
+
+    # If too many arguments are supplied, return an error
+    if sum(arg is not None for arg in [ip, network, from_ip and to_ip]) > 1:
+        raise ValueError("Please specify only one of the `ip`, `network` or `from_ip`/`to_ip` arguments")
+
+    # If neither ip, network nor from/to_ip were specified, return an error.
+    elif sum(arg is not None for arg in [ip, network, from_ip and to_ip]) == 0:
+        raise ValueError("Please specify either the `ip`, `network` or `from_ip`/`to_ip` argument")
+
+    extended_attributes = args.get("extended_attrs")
+    if extended_attributes:
+        client.set_param(client.REQUEST_PARAM_EXTRA_ATTRIBUTES)
+
+        extended_attributes_params = transform_ext_attrs(extended_attributes)
+
+        for e in extended_attributes_params:
+            client.set_param(e)
+
+    max_results = arg_to_number(args.get('max_results', INTEGRATION_MAX_RESULTS_DEFAULT), required=False)
+    client.set_param({InfoBloxNIOSClient.REQUEST_PARAM_MAX_RESULTS_KEY: max_results})
+
+    # Check if the network/IPs supplied are valid.
+    if ip:
+        valid_ip(ip)
+        status = args.get('status', IPv4AddressStatus.USED.value)
+        raw_response = client.get_ipv4_address_from_ip(ip, status=status)
+    elif network:
+        valid_netmask(network)
+        status = args.get('status', IPv4AddressStatus.USED.value)
+        raw_response = client.get_ipv4_address_from_netmask(network, status=status)
+    elif from_ip and to_ip:
+        valid_ip_range(from_ip, to_ip)
+        raw_response = client.get_ipv4_address_range(from_ip, to_ip)
+
     ip_list = raw_response.get('result')
 
-    # If no IP object was returned
     if not ip_list:
-        return f'{INTEGRATION_NAME} - Could not find any data corresponds to: {ip}', {}, {}
-    fixed_keys_obj = {RESPONSE_TRANSLATION_DICTIONARY.get(key, string_to_context_key(key)): val for key, val in
-                      ip_list[0].items()}
-    title = f'{INTEGRATION_NAME} - IP: {ip} info.'
-    context = {
-        f'{INTEGRATION_CONTEXT_NAME}.IP(val.ReferenceID && val.ReferenceID === obj.ReferenceID)': fixed_keys_obj}
-    human_readable = tableToMarkdown(title, fixed_keys_obj, headerTransform=pascalToSpace)
+        human_readable = f'{INTEGRATION_NAME} - Could not find any data'
+        context = {}
+    else:
+        output = transform_ip_context(ip_list)
+        title = f'{INTEGRATION_NAME} - IP info (limit {max_results})'
```

Comment:
Because the `ip` argument is no longer mandatory, the command can be run with either `ip`, `network` or `from_ip` to `to_ip` arg. Maybe what I can do is set the title earlier in the `if..else` block where i check with argument was specified.

Reply:
set arg in title.